### ISSUE TITLE
Is there a way to specify the scheme to use on a request?

### ISSUE THREAD

**Issue by MartinDelVecchio** [mission-liao/pyswagger#133]
The swagger I am testing specifies two schemes:

schemes:
- http
- https

But the system I am testing against only actually supports HTTPS.  I recognize that this is a bug, or invalid swagger.

But I am hoping to work around it by specifying HTTPS for each request I make.  I am executing a request with this call:

        # Execute it
        response = self.client.request (self.app.op[operation_id](**kwargs), opt=opt, headers=headers)

I would love to be able to specify the scheme either in the opt dictionary (where I cam currently specifying url_netloc.  For example:

    opt = {'url_netloc': self.netloc, 'url_scheme': 'https'}

Is there a way to do this now that I am not seeing?

Thanks!

**Comment by mission-liao** [comment]
The way you suggested is fine, I'll add it in next release. Before I add that capability. You can restrict the scheme used when making requests by inheriting the client class, ex:

```python
from pyswagger.contrib.client.requests import Client

class YourCustomizedClient(Client):
    __schemes__ = ['https']

# use your customized client for later request
```

**Comment by MartinDelVecchio** [comment]
Thanks, I will try that this weekend.

Thanks.

**Comment by mission-liao** [comment]
This issue should be fixed in `v0.8.35`, please feel free to open it if it's not working.

**Comment by MartinDelVecchio** [comment]
Verified.  Thanks!
